src/shipment/controller.js

```diff
@@ -393,6 +396,10 @@ async function deleteOne(req, res) {
       );
 
       await Promise.all([p_delete, p_updatePackingSlips]);
+
+      await IncomingDelivery.findOneAndDelete({
```

Comment:
Use `deleteOne` instead.
In this case, `deleteMany` would be better, since we might allow for multiple incomings per outgoing.

Reply:
Changed to deleteMany.